core/modules/savers/put.js

```diff
@@ -25,8 +25,9 @@ var RetrieveETag = function(self) {
 		type: "HEAD",
 		headers: headers,
 		callback: function(err, data, xhr) {
-			if(!err) {
-				self.etag = xhr.getResponseHeader("ETag").replace(/^W\//,"");
+			var etag = xhr.getResponseHeader("ETag");
+			if(!err && etag) {
+				self.etag = etag.replace(/^W\//,"");
```

Comment:
This crashes because when the page first boots callback is called with an error which mean `xhr` is undefined.

#3261 is opened for this bug.

Reply:
Fixed in PR #3262

Reply:
Why is there an error at boot? Which setup did you use/test?